pr2eus/robot-interface.l

```diff
@@ -251,6 +251,11 @@
 
   (:angle-vector-duration
    (start end &optional (scale 1) (min-time 1.0) (ctype controller-type))
+   (when (or (null start) (null end))
```

Comment:
when this situation happens.
https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus/robot-interface.l#L304 ? do we set av = nil?

https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus/robot-interface.l#L357 potentio-vector (https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus/robot-interface.l#L347) returns nil?

https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus/robot-interface.l#L372 avs and tms has different length